### PR TITLE
Remove marker example from parsefunc.short

### PR DESCRIPTION
**Pull request by froemken**

### CODE REVIEW

File: Documentation/Functions/Parsefunc.rst [hunk 1]
```diff
@@ -119,23 +119,22 @@ short
 -----
 
 ..  confval:: short
-    :name: arsefunc-short
```
Comment (garvinhicking) [inline Documentation/Functions/Parsefunc.rst:122]: LOL, really? :-D

**Review by garvinhicking — COMMENTED** [review]
(sadly can't comment on the actual change here, too little integrator knowledge...)

File: Documentation/Functions/Parsefunc.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/Functions/Parsefunc.rst:122]: :-D

**Comment by linawolf** [comment]
@froemken is there a changelog entry on when the behaviour changed?

**Comment by froemken** [comment]
@linawolf It has never be changed. I just go back until TYPO3 4.5 and it has never been marker-based:
https://github.com/TYPO3/typo3/blob/TYPO3_4-5/typo3/sysext/cms/tslib/class.tslib_content.php#L4540-L4547

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
Ok, then we documented this wrong in the first place

**Pull request merged**